helm/templates/sessions/sessions.configmap.yaml

```
@@ -64,46 +64,138 @@ data:
     <!DOCTYPE html>
     <html>
       <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <style>
           body {
-            font-family: Arial, sans-serif;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI",
+              Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue",
+              sans-serif;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            background-color: #000;
+            color: white;
+            margin: 0;
+            height: 100vh;
+            justify-content: center;
```

```suggestion
            display: grid;
            place-items: center;
            background-color: #000;
            color: white;
            margin: 0;
            height: 100vh;
```
We can simplify the centering using `grid` and `place-items: center`. As there's only one element in the flexbox the `flex-direction` is not necessary.

---

Thanks for the hint. It also makes the `viewport` definition unnecessary.